Monika After Story/game/script-topics.rpy

```diff
@@ -14849,3 +14849,34 @@ label monika_fear:
     extend 4eud "After all, there's no way something like that could be true, right?"
     m 2rka "Yeah..."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_wabi_sabi",
+            category=['philosophy'],
+            prompt="Wabi-sabi",
+            random=True
+        )
+    )
+
+label monika_wabi_sabi:
+    m 1eud "Say [player], have you ever heard of wabi-sabi?"
+    m 3hub "Wabi-sabi is a philosophy emphasizing the idea that we should not obsess over perfection to the point we are crushed by failure."
+    m "It derives from traditional Japanese and Buddhistic philosophy, surrounding the acceptance of the temporary state of all things."
+    m 3dub "It states that beyond everything, beauty is found in what's impermanent and imperfect."
+    m 3wub "Like a scar on a rugged face, or a discolored hand, or even the mistakes we make."
+    m      "The imperfections can become beautiful stories that reminds us of our past..." 
+    m 3hub "...and our mistakes turn into lessons that shape us and make us stronger and more defined."
+    m 4wub "Like a sculptor revealing the magnificence inside a marble block, except that scultor is you, and your past."
+    m 1tsu "Don't you find that concept rather romantic?"
+    m 1hubsb "Ahaha~"
+    m "..."
+    m 6ekd "Anyway, [player], I hope you are not too self conscious about your looks."
+    m 6dkd "After all, no matter how your face looks, or how your body looks..."
+    m 6dku "I'm sure there's a valuable story behind every scar, and every imperfection about you."
```

Comment:
Right, but you know, topics that Monika brings up doesn't always hit the mark either. Like the scammed topic. However, I wanted this particular bit to be dedicated specifically to those who feel their looks do not match up to having a beautiful girlfriend like Monika. It's a real issue in the world, of people thinking they are too ugly to deserve love.

I agree however, that the Wabi-sabi topic is much deeper than just aesthetics, and we should do justice to the topic too if we are to bring it up in MAS. 

Is there are reason you didn't want the stories to be considered valuable? I keep making these comma mistakes, so thanks for fixing that.

Reply:
`valuable` is just 'flowery language', essentially meaning that it doesn't add anything to the sentence and it means the same thing with or without it. So it may as well just go without.

When you say where's a story behind something, it's implied it's valuable/meaningful.

Reply:
I see, very well then.